src/index.js

```diff
@@ -26,6 +27,18 @@ import {
   NoteEditPage
 } from './views';
 import KeyboardShortcutsModal from './components/KeyboardShortcutsModal';
+import { PatronBlockMessage } from './components/PatronBlock';
+
+const modifiedCommands = [...commands,
+  {
+    name: 'nextPane',
+    shortcut: 'shift + pagedown'
```

Comment:
FYI, no `pagedown` or `pageup` keys on a MacOS laptop keyboard. I know I know, WIP, POC, etc.

Reply:
Thank you for re-iterating this on this rather ancient POC... For persistence of the conversation, Does `fn up/down arrow` perform the same function/key as pageup/down? can you `shift + fn + arrow` on a mac?